plugin/src/main/res/values/styles.xml

```diff
@@ -24,4 +24,7 @@
         <item name="android:windowCloseOnTouchOutside">false</item>
         <item name="windowNoTitle">true</item>
     </style>
+    <style name="AlertDialogTheme" parent="Theme.MaterialComponents.DayNight.Dialog.Alert">
```

Comment:
`ThemeOverlay` should probably be used instead to not screw up the color scheme.